src/core/server/metrics/collectors/cgroup.ts

```diff
@@ -64,12 +66,11 @@ export class OsCgroupMetricsCollector implements MetricsCollector<OsCgroupMetric
         },
       };
     } catch (err) {
-      if (err.code === 'ENOENT') {
-        this.noCgroupPresent = true;
-        return {};
-      } else {
-        throw err;
```

Comment:
Does it comply with the existing LP logic?

Reply:
Well, there is no more legacy logic :) But yes, this matches the catch-all exception handler that was at the calling level here: https://github.com/elastic/kibana/blob/7.8/src/legacy/server/status/lib/metrics.js#L131